futures-util/src/async_await/join.rs

```diff
@@ -1,5 +1,25 @@
-//! The `join` macro.
 
+/// The `join` macro.
```

Comment:
This text was a summary for the entire module. It's not required because the module isn't exported publicly. Now, you moved into the description for `join!`. Usually descriptions don't start with a headline, but with a summary sentence.